securedrop/crypto_util.py

```diff
@@ -315,13 +310,11 @@ def get_pubkey(self, name):
         self.redis.hset(self.REDIS_KEY_HASH, fingerprint, key)
         return key
 
-    def encrypt(self, plaintext, fingerprints, output=None):
+    def encrypt(self, plaintext: str, fingerprints: List[str], output: Optional[str] = None) -> str:
         # Verify the output path
         if output:
             current_app.storage.verify(output)
 
-        if not isinstance(fingerprints, (list, tuple)):
-            fingerprints = [fingerprints, ]
```

Comment:
I changed the callers to always pass a list

Reply:
:+1: